js/src/ui/StatusIndicator/statusIndicator.js

```diff
@@ -37,6 +37,11 @@ export default class StatusIndicator extends Component {
 
   render () {
     const { id, status, title, type, tooltipPlacement } = this.props;
+    const tooltip = title.find(x => !x.isEmpty) ? (
```

Comment:
I could not get the tooltip to disable, even by setting the `data-tip` explicitly to `false` or `''`.